apps/coscrad-frontend/src/components/tenant/tenant-logo.tsx

```diff
@@ -6,10 +7,13 @@ export const TenantLogo = (): JSX.Element => {
     const { organizationLogoUrl } = useContext(ConfigurableContentContext);
 
     return (
-        <StyledImage
-            sx={{ width: { xs: '40px', sm: '55px', md: '61px' } }}
-            src={organizationLogoUrl}
-            alt="COSCRAD Logo"
-        />
+        //TODO parent organization url here
+        <IconButton>
```

Comment:
I forgot all about this. You might want to use this approach. However, I wonder if you’d have more luck if the logo had to be an SVG?